### PR TITLE
Add download command

### PR DESCRIPTION
**Pull request by arturdm**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  - stable
+  - dev
+
+cache:
+  directories:
+    - $HOME/.pub-cache
+    - .dart_tool/build
+
+before_install:
+  - git clone https://github.com/flutter/flutter.git -b stable $HOME/flutter
```
Comment (WezSieTato) [inline .travis.yml:12]: i'm wonder when travis will be support flutter like this:
```
language: flutter
flutter_version: 1.13
```

File: lib/src/client/lokalise_client.dart [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+          _xApiTokenHeader: _apiToken,
+        },
+        body: jsonEncode(requestBody.toJson()));
+    return LokaliseResponse.from(response, (json) => FilesDownloadResponseBody.fromJson(json));
+  }
+}
+
+typedef FromJson<T> = T Function(Map<String, dynamic>);
+
+class LokaliseResponse<T> {
```
Comment (WezSieTato) [inline lib/src/client/lokalise_client.dart]: New file for this? When I saw LokaliseResponse, as a output of public method I started searching it in separate file.

File: lib/src/pubspec_config/pubspec_config.dart [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:yaml/yaml.dart';
+
+part 'pubspec_config.freezed.dart';
+
+@freezed
+abstract class PubspecConfig implements _$PubspecConfig {
```
Comment (WezSieTato) [inline lib/src/pubspec_config/pubspec_config.dart]: `LokaliseConfig`? In factory method we have information that this is from pubspec. In my opinion this is implementation detail at this level.

File: pubspec.yaml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+environment:
+  sdk: ">=2.6.0 <3.0.0"
+
+executables:
+  flutter_lokalise:
+
+dependencies:
+  archive: ^2.0.11
+  args: ^1.5.2
+  equatable: ^1.1.1
```
Comment (WezSieTato) [inline pubspec.yaml]: Never used :)

File: lib/src/pubspec_config/pubspec_config.dart [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    String projectId,
+    String apiToken,
+    Iterable<String> includeTags,
+    String output,
+  }) = _PubspecConfig;
+
+  factory PubspecConfig.fromPubspecYamlString(String yamlString) {
+    final yaml = loadYamlNode(yamlString);
+    final flutterLokalise = yaml.getOrNull<YamlMap>("flutter_lokalise");
+    return PubspecConfig(
```
Comment (WezSieTato) [inline lib/src/pubspec_config/pubspec_config.dart]: Maybe you can return nil when flutter_lokalise doesn't exist in pubspec?

**Review by WezSieTato — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (arturdm) [reply .travis.yml:12]: me too :)

File: lib/src/pubspec_config/pubspec_config.dart (hunk 5 above)
Comment (arturdm) [reply lib/src/pubspec_config/pubspec_config.dart]: I left it as it was - it doesn't provide much info right now.

**Review by WezSieTato — APPROVED** [review]

**Pull request merged**